k8s/spinnaker/README.md

```diff
@@ -10,97 +10,117 @@ If you would like to learn more about Spinnaker, please visit the
+
+When the initial installation process completes, additional commands will be shown
+to allow you to:
 
-When you click on the "Open in Google Cloud Shell" button above, a tutorial
-pane will open with several copy/paste commands. As part of this process, a
-`properties` file will be generated. You can modify the values in the generated
-`properties` file to fine-tune the installation _prior_ to launching the actual
-`setup` script.
+* find your Spinnaker deployment in the GKE Applications view
+* access Spinnaker via port-forwarding
+* expose Spinnaker securely using IAP
+* allow others to administer and/or access Spinnaker
 
-When the initial installation process completes, you can use the remaining
-copy/paste commands to locate your Spinnaker deployment in the GKE Applications
-view, access Spinnaker via port-forwarding, expose Spinnaker securely, share
-Spinnaker with your teammates, and so on.
+and more.
 
 #### Ongoing Management of Spinnaker
 
-The ongoing management workflow will be as follows:
+To manage your installed Spinnaker instance:
 
 * Locate your Spinnaker installation by navigating to the newly-registered
-Kubernetes Application via the
-[Applications](https://console.developers.google.com/kubernetes/application?project=_)
+Application in the
+[GKE Applications](https://console.developers.google.com/kubernetes/application?project=_)
 view.
-* Click the link from the Spinnaker Application to open Cloud Shell and clone
-this repo
-* Ensure you are connected to the correct GKE cluster (that is, the one
-containing the Spinnaker deployment you intend to manage)
-* Pull all the config from the Spinnaker deployment into your Cloud Shell
-environment
-* Update the Cloud Shell tutorial pane to reflect the newly-pulled config
-* Make all of your changes using `hal` in Cloud Shell (often by simply
-copying & pasting commands); `hal` will apply its changes and perform its
-validation against a Halyard daemon running in Cloud Shell
-* Push your changes back to the Spinnaker deployment (a full config backup is
-implicitly pushed to a bucket as well)
-* Apply your changes to the Spinnaker deployment
-
-There are scripts provided for each of these steps, and copy/paste commands and
-instructions are rendered in the Cloud Shell tutorial pane.
-
-This approach allows for an Admin to manage multiple Spinnaker deployments, as
-the source of truth for the deployment's configuration is always the deployment
-itself. It also allows for easily sharing management of Spinnaker among
-multiple Admins.
+* Click the link from the Spinnaker Application to open the management
+environment in Cloud Shell, and update it by cloning this repo
+* make sure you're
+connected to the GKE cluster containing the Spinnaker instance you want to manage.
+* Update your configuration by pulling the config from the Spinnaker instance into your
+management environment.
+* Refresh the management environment interface to reflect your updated configuration.
```

Comment:
@dorbin Perhaps we should say "Update the..." here to avoid potential confusion that this implies reloading/refreshing the browser?